CIP-0XXX/README.md

```diff
@@ -0,0 +1,540 @@
+---
+CIP: ?
+Title: Cardano Decentralized Message Queue
+Category: ?
+Status: Proposed
+Authors:
+    - Jean-Philippe Raynaud <jp.raynaud@gmail.com>
+    - Arnaud Bailly <arnaud.bailly@iohk.io>
+    - Marcin Szamotulski <marcin.szamotulski@iohk.io>
+    - Armando Santos <armando.santos@iohk.io>
+    - Neil Davies <neil.davies@iohk.io>
+    - Sebastian Nagel <sebastian.nagel@ncoding.at>
+    - ?
+Implementors: 
+    - Cardano Scaling team <https://github.com/cardano-scaling>
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/?
```

Comment:
```suggestion
    - https://github.com/cardano-foundation/CIPs/pull/876
```
(this discussion)

Reply:
Done in https://github.com/cardano-scaling/CIPs/commit/e154a146fb1ad42d98616814b28048b39d14ea71